### DISCUSSION TITLE
Help with configuring package.json (PostCSS + Tailwind 3)

### DISCUSSION THREAD

**Discussion by Sagefully** [tailwindlabs/tailwindcss#6511] [Help]
I am following the guide here: https://tailwindcss.com/docs/installation/using-postcss

However, the guide may be simple but it is not kind to noobs such as I. 

1. The guide does not specify that I have to create a postcss.config.js file myself (if I do not already have it, which I don't, since I am starting from nothing).
2. Following the guide: Step 5 tells me to "[...] run npm run dev or whatever is configured in your package.json". Thing is, when I followed the guide, I never configured package.json. I do not know how to configure package.json. Following the guide, all there is in the package.json is `{
  "devDependencies": {
    
    "autoprefixer": "^10.4.0",
    "postcss": "^8.4.5",
    "tailwindcss": "^3.0.2"
  }

}`

So I guess my question and call for help is regarding package.json. How do I configure it so that my terminal can happily proceed?

**Comment by lukewarlow** [comment]
Please provide more information about your setup. Presumably if you don't already have a package.json you were not previously using node in your project?
